api/db-cdr/generate-cdr/build-cb-criteria-attribute-tables-and-cleanup.sh

```diff
@@ -375,4 +375,21 @@ echo "CLEAN UP - remove any survey with zero est_count"
 bq --quiet --project_id="$BQ_PROJECT" query --batch --nouse_legacy_sql \
 "DELETE \`$BQ_PROJECT.$BQ_DATASET.cb_criteria\`
 WHERE domain_id = 'SURVEY'
-AND est_count = 0"
\ No newline at end of file
+AND est_count = 0"
+
+echo "CLEAN UP - remove any empty survey topics"
```

Comment:
Adding delete statement to clean up any empty topics since we are inserting all topics

Reply:
Didn't think of  that one... but got the one at #206 for buid-prep-survey.sh